carapp/__init__.py

```diff
@@ -24,11 +29,9 @@
 Database migrations.
 """
 
-migrate = Migrate(app, db)
+migrate = Migrate(app, db, compare_type=True)
```

Comment:
Overall it is better to have a management command to migrate. It is out of the PR scope, so let's have a separate ticket for the improvement.

Reply:
Yes sure. I have used command to migrate but it was necessary to add this change(**compare_type=True**) to update password credentials in DB (unfortunately, flask-migration can't catch such changes without this manipulation).